packages/mocks/src/server.ts

```diff
@@ -0,0 +1,44 @@
+import { readFileSync } from 'fs';
+import process from 'process';
+
+import express from 'express';
+import expressHttpProxy from 'express-http-proxy';
+import minimist from 'minimist';
+
+import { createMiddleware } from '@mswjs/http-middleware';
+
+import { createDefaultHandlers } from './handlers';
+import defaultPrefixMap from './prefixMap.json';
+import { createHandlersFromHar } from './utils';
+
+const argv = minimist(process.argv.slice(2));
+
+const port = argv['port'] || 30088;
+const host = argv['host'] || `http://localhost:${port}`;
+const harFile = argv['harFile'] || undefined;
+const prefixMapFile = argv['prefixMapFile'] || undefined;
+
+const server = express();
+server.disable('x-powered-by');
+const proxy = expressHttpProxy(host);
```

Comment:
So the `host` is the host being proxied?  Then the default value with the `port` included doesn't make sense.  Maybe call it `proxyHost` instead?

Reply:
The defaults match the `INVENTORY_SERVER_HOST` see -> https://github.com/kubev2v/forklift-console-plugin/blob/d0a32c380461dd3315070bb1fd6165e7c92786d6/ci/start-console.sh#L29

Reply:
Humm.  In that case, it should be called that.  No fun to have generic terms like "host" for a specific kind of host to be man-in-the-middled (er I mean mocked)

Reply:
Let us decide later on about the naming - the more precise alternatives i.e. proposed above `proxyHost` or `sourceHost` or similar are a bit verbose.